### PR TITLE
feat: Add Report Page

### LINKED ISSUE #12: [Report] View report
image

### LINKED ISSUE #13: [Report] Add Report Page
image

### PR DESCRIPTION
**Pull request by SpottedYodeler2**
**Add Report Page accessible at /Report**

A modal will appear when plus button is clicked in the add report page. 

Closes #12
Closes #13

### CODE REVIEW

**Comment by SpottedYodeler2** [comment]
Demo:

https://github.com/user-attachments/assets/e17df8ca-2beb-4aad-9371-542325ca0f16

File: src/App.jsx [hunk 1]
```diff
@@ -6,13 +6,15 @@ import NotFound from "./pages/NotFound";
...
 import "./App.css";
+import AddReportPage from "./pages/AddReportPage";
 
 function App() {
   return (
     <Routes>
       <Route path="/" element={<Layout />}>
         <Route index element={<Home />} />
         <Route path="report" element={<ReportPage />} />
+        <Route path="addreport" element={<AddReportPage />} />
```
Comment (chaaals) [inline src/App.jsx]: we're opening a modal, looks like hindi na natin need ito

File: src/pages/AddReportPage.jsx
Comment (chaaals) [inline src/pages/AddReportPage.jsx:1]: I think we can remove this kasi we're adding a modal

**Review by chaaals — CHANGES_REQUESTED** [review]
lgtm on UI! may comment lang, correct me if I'm wrong

File: src/App.jsx (hunk 1 above)
Comment (uzuriel) [reply src/App.jsx]: i agree with this one, di na need magredirect sa ibang url

Comment (SpottedYodeler2) [reply src/App.jsx]: ay yeah I used that on my early testing lang

File: src/pages/AddReportPage.jsx
Comment (SpottedYodeler2) [reply src/pages/AddReportPage.jsx:1]: oki will do

**Review by uzuriel — APPROVED** [review]

**Review by chaaals — APPROVED** [review]

**Pull request merged**